package.json

```diff
@@ -75,5 +75,8 @@
   "workspaces": [
     "./packages/*",
     "scripts/generate-docs-index"
-  ]
+  ],
+  "dependencies": {
+    "browserslist-config-godaddy": "^3.0.1"
```

Comment:
This is an internal package, no? Probably shouldn't be a dependency of this open source package.